rolldown.ts

```diff
@@ -10,7 +10,11 @@ export const defineConfig = (options: RolldownOptions): RolldownOptions => {
     {
       platform: 'browser',
       plugins: [dts({ resolve: true })],
-      external: Object.keys(pkg.dependencies ?? {}).flatMap((k) => [k, new RegExp(`^${k}/`)]),
+      external: [
+        'yaml-to-momoa',
```

Comment:
I feel like this ought to be an optional peer dependency in the places it's currentl referenced...but for now, just marked it as always external

Reply:
Oh interesting! I’ve never tried optional peer dependencies but that seems correct.

Reply:
It's a weird one because it _will_ install it (unless you specifiy no optional in your package manager)...it'll just be ok if it fails to install for some reason. Moving it to peerDeps will mean it'll get _installed_ regardless, I think....at which point, why not just have it a dep?

[Python](https://peps.python.org/pep-0508/#extras) and [Rust](https://doc.rust-lang.org/cargo/reference/features.html) ahead of the game, again...

Reply:
For more context, the reason `yaml-to-momoa` is a separate, manual install for `@terrazzo/parser` is for those few users who are building client-side tools, `yaml-to-momoa` introduces the `yaml` dep which is extremely heavy. For Node CLIs, though, it’s trivial, and we install it automatically with `@terrazzo/cli`.

To your point, though, we could just install it automatically and that would be fine, since they still have to manually load `yaml-to-momoa` in the parser API. So I wouldn’t mind it being a dep or peerDep that gets automatically installed.

Reply:
Cool, I'll convert it to a peer dep and see what happens (: